manifests/0000_09_cluster-authentication-operator_05_deploy.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  namespace: openshift-core-operators
-  name: origin-cluster-osin-operator
+  namespace: openshift-authentication-operator
```

Comment:
do not mess with the original operator, we have no tests

Reply:
ok, done